Draft of "Who Owns Contenda content?" (#29)
Contenda-Team/Contenda-landing-page: src/posts/whoownscontent.md

---
layout: "../layouts/BlogPost.astro"
title: "Who owns my Contenda-generated content?"
slug: who-owns-content
description: "The rules-- social and legal-- are murky. Can I say I wrote it?"
author: "Lilly Chen"
added: "April 19 2023"
tags: ["opinion"]
---

> I most commonly see this debate amongst AI-generated art and code. If CoPilot uses copywrited code, can someone be sued for using it? If someone deliberately tries to use AI to generate artwork from an artist, is that stealing? 

Reviewer: ```suggestion
I most commonly see this debate amongst AI-generated art and code. If GitHub Copilot uses copyrighted code, can someone be sued for using it? If someone deliberately tries to use AI to generate artwork from an artist, is that stealing? 
```

## Can I say I wrote Contenda-content?

If you are the original video creator, you should absolutely say you wrote it! Contenda would do nothing without your excellent narration. 

> If you are not the original video creator, we **highly encourage** you to cite the video. For brownie points, send the draft of Contenda's blog to the original video creator for a proofread. 

Reviewer: I feel like this is incredibly hairy. We should almost say that you should own the videos/have the rights to mess with videos that you process?

"If you are not the original video creator, make sure that you have the permissions or rights to work with that creator's video, whether they're on your team or a more video-savvy friend. None of us want to steal from others, here!"

...or something.

> 

Reviewer: just removing the extra line break

## What if I add significant changes to a Contenda blog?

Let's say you ingested multiple videos from different creators about the same topic. Then you combined the Contenda blogs to create a net new blog about the hot takes of topic X. In that context, you should still cite the original videos, but you are the author of this new shiny blog post! 

> ## My situation is somewhere inbetween

Reviewer: ```suggestion
## My situation is somewhere in-between.
```

When in doubt, give credit where credit is due. If someone asks you nicely to change something, maybe it's a good suggestion overall. 

> 

Reviewer: ```suggestion
```

removing line break
